Frontend/frontend/src/Components/Alert.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import "./Alert.css";
+
+export default class Alert extends React.Component {
```

Comment:
Alert should be renamed to something that more accurately reflects it's use case like `ConfirmRemoveFlatMember` (wow that is a bit long tho, see if you can think of anything better haha but it's also fine).